.tool-versions

```diff
@@ -1,3 +1,3 @@
 rust 1.26.0
-elixir 1.7.2
-erlang 20.0
+elixir 1.7.2-otp-21
```

Comment:
I've updated these to match what we have on circle to try and reproduce locally running dialyzer without emitting any warnings.

Reply:
I think it's wrong because we can now only use otp 20

Reply:
til https://github.com/urbint/rox/pull/15 is merged

Reply:
The weird thing is that CI uses otp-21 and all the tests pass. I guess the tests are using `ets` instead of `rox`?

Reply:
yep @germsvel